backend/fms_core/viewsets/_constants.py

```diff
@@ -153,20 +153,7 @@
 _run_type_filterset_fields: FiltersetFields = {
     "id": PK_FILTERS,
     "name": CATEGORICAL_FILTERS_LOOSE,
-}
-
-_experiment_run_filterset_fields: FiltersetFields = {
-    "id": PK_FILTERS,
-    "name": CATEGORICAL_FILTERS_LOOSE,
-    "start_date": DATE_FILTERS,
-    "run_type": FK_FILTERS,
-    "instrument": FK_FILTERS,
-    "container": FK_FILTERS,
-    "run_processing_launch_time": DATE_FILTERS,
-
-    **_prefix_keys("run_type__", _run_type_filterset_fields),
-    **_prefix_keys("instrument__", _instrument_filterset_fields),
-    **_prefix_keys("container__", _container_filterset_fields),
+    **_prefix_keys("platform__", _platform_filterset_fields),
```

Comment:
Run type name might actually be a better identifier for the grouping ... You could display Illumina, Axiom, DNBSEQ, Infinium straight  out of the DB. You can leave the filterset field for the platform but it might actually be one additional indirection we might not need.